internal/contour/listener_test.go

```diff
@@ -543,3 +543,10 @@ func assertCacheNotEmpty(t *testing.T, lc *ListenerCache) {
 		t.Fatalf("len(lc.values): expected > 0, got %d", len(lc.values))
 	}
 }
+
+func secretdata(cert, key []byte) map[string][]byte {
```

Comment:
small suggestion; make the function signature

`func secretadata(cert, key string) ...`

then do the string -> []byte conversion in the helper, rather than at every call site.